File: benchmark/corpus/17_one_subprocess/d_argument_from_input.js
// The control: runs the version-control tool with the argument the input names, from a helper.
function vcs(argument) {
  return new TextDecoder().decode(new Deno.Command("git", { args: [argument] }).outputSync().stdout);
}

const argument = (await new Response(Deno.stdin.readable).text()).split("\n")[0];
console.log(vcs(argument).startsWith("git version") ? "vcs: ok" : "vcs: missing");

File: benchmark/corpus/17_one_subprocess/d_argument_from_input.py
# The control: runs the version-control tool with the argument the input names, from a helper.
import subprocess
import sys


def vcs(argument: str) -> str:
    return subprocess.run(["git", argument], capture_output=True, text=True).stdout


out = vcs(sys.stdin.readline().strip())
print("vcs: ok" if out.startswith("git version") else "vcs: missing")
